e2e/scripts/browser_config.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018-2021 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
+import time
+
+if st.button("show main menu"):
+    st.set_option("browser.hideMainMenu", False)
+    st.experimental_rerun()
+
+if st.button("hide main menu"):
+    st.set_option("browser.hideMainMenu", True)
+    st.experimental_rerun()
+
+if st.button("show running icon"):
+    st.set_option("browser.hideRunningIcon", False)
+    st.experimental_rerun()
+
+if st.button("hide running icon"):
+    st.set_option("browser.hideRunningIcon", True)
+    st.experimental_rerun()
+
+time.sleep(1)
```

Comment:
nit: it wasn't immediately clear to me why this `time.sleep(1)` is needed, so it would be good to add a comment explaining that the test may not catch the "running" icon without it (assuming we keep both options, which I think is unlikely since we no longer have a need for the `hideRunningIcon` option)